tests/MongoDB.Driver.Legacy.Tests/Builders/IndexKeysBuilderTests.cs

```diff
@@ -70,7 +70,14 @@ public void CreateIndex_with_wildcardProjection_should_create_expected_index()
             var index = indexes.RawDocuments.Single(i => i["name"].AsString == "custom");
 
             index["key"]["$**"].AsInt32.Should().Be(1);
-            index["wildcardProjection"].Should().Be(BsonDocument.Parse("{ _id : 1 }"));
+            if (CoreTestConfiguration.ServerVersion >= new SemanticVersion(4, 5, 0, ""))
+            {
+                index["wildcardProjection"].Should().Be(BsonDocument.Parse("{ _id : true }"));
+            }
+            else
+            {
+                index["wildcardProjection"].Should().Be(BsonDocument.Parse("{ _id : 1 }"));
+            }
```

Comment:
This change isn't relate to THIS ticket is it?

We could just include it... or it could be a separate ticket.

Reply:
it's cherry-picked from master

Reply:
OK